<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::MainWindow</class>
 <widget class="QMainWindow" name="Ripes::MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="centrallayout">
      <item>
       <widget class="FancyTabBar" name="tabbar" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_wiki"/>
    <addaction name="actionSystem_calls"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_wiki">
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>About Ripes</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSystem_calls">
   <property name="text">
    <string>System calls</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FancyTabBar</class>
   <extends>QWidget</extends>
   <header>fancytabbar/fancytabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
